pm/sender.go

```diff
@@ -74,13 +75,31 @@ func (s *sender) EV(sessionID string) (*big.Rat, error) {
 	return ticketEV(session.ticketParams.FaceValue, session.ticketParams.WinProb), nil
 }
 
+func (s *sender) validateSender() error {
+	info, err := s.senderManager.GetSenderInfo(s.signer.Account().Address)
+	if err != nil {
+		return fmt.Errorf("unable to validate sender: could not get sender info: %v", err)
+	}
+
+	maxWithdrawRound := new(big.Int).Add(s.roundsManager.LastInitializedRound(), big.NewInt(1))
+	if info.WithdrawRound != nil && info.WithdrawRound.Int64() != 0 && info.WithdrawRound.Cmp(maxWithdrawRound) != 1 {
```

Comment:
Don't think `info.WithdrawRound` will ever be nil

Reply:
fixed in 4fb8b5d2

Reply:
actually this is nil in most of the tests unless we set WithdrawRound in every test in `sender_test`  instead of simply using `defaultSender` helper so would you be ok with keeping it there?

Reply:
fixed in 915b0de3